tests/snapshots/files__implicit-return-optimize.snap

```diff
@@ -53,31 +54,32 @@ expression: visualization.result
   v6_: int = id c1_;
   v7_: int = id c3_;
 .b8_:
-  c9_: int = const 14;
```

Comment:
huh, I wonder why the old one has const 14 while the new one doesn't

Reply:
seems like we missed the constant folding and computed c9_ as 15 - 1 (v10_ in the new program)

Reply:
That does seem like a regression- do you know why it isn't happening now? Perhaps we actually did need `Get` for a generated concat?
Could be that we need to demand `Get` for all of the generated `Concat`, just not subtuples

Reply:
oh nice this is gone after your new IVT is merged